client/packages/invoices/src/utils.ts

```diff
@@ -156,6 +174,18 @@ export const isInboundDisabled = (inbound: InboundRowFragment): boolean => {
         inbound.status === InvoiceNodeStatus.Verified;
 };
 
+export const isInboundReturnDisabled = (
+  inboundReturn: InboundReturnRowFragment
+): boolean => {
+  // const isManuallyCreated = !inbound.linkedShipment?.id;
```

Comment:
Do we have an issue to handle the auto-linked invoices?

Reply:
We have one for "creating the other half with processors" but I'll add to it to include reviewing how we display manually created vs automatically created inbound returns 👍